pyproject.toml

```diff
@@ -285,7 +285,6 @@ asyncio_default_fixture_loop_scope = "class"
 python_version = "3.9"
 disallow_incomplete_defs = true
 warn_return_any = false
-warn_unused_configs = true
```

Comment:
enabling this argument without setting `incremental = False` does not work ([mypy docs](https://mypy.readthedocs.io/en/stable/config_file.html#confval-warn_unused_configs))

And [incremental mode](https://mypy.readthedocs.io/en/stable/command_line.html#incremental) is useful for faster running mypy locally